homeassistant/components/denonavr/__init__.py

```diff
@@ -23,32 +22,12 @@
 
 CONF_RECEIVER = "receiver"
 UNDO_UPDATE_LISTENER = "undo_update_listener"
-SERVICE_GET_COMMAND = "get_command"
 
 _LOGGER = logging.getLogger(__name__)
 
-CALL_SCHEMA = vol.Schema({vol.Required(ATTR_ENTITY_ID): cv.comp_entity_ids})
-
-GET_COMMAND_SCHEMA = CALL_SCHEMA.extend({vol.Required(ATTR_COMMAND): cv.string})
-
-SERVICE_TO_METHOD = {
-    SERVICE_GET_COMMAND: {"method": "get_command", "schema": GET_COMMAND_SCHEMA}
-}
-
 
-def setup(hass: core.HomeAssistant, config: dict):
+async def async_setup(hass: core.HomeAssistant, config: Dict):
```

Comment:
`async_setup` is no longer required when `async_setup_entry` exists as of the latest `upstream/dev` 

If you rebase, you should now be able to remove it.